### DISCUSSION TITLE
Context menu (right click menu) changed order

### DISCUSSION THREAD

**Discussion by tomiesz** [qutebrowser/qutebrowser#5716] [Q&A] [answered]
I noticed today that the context menu order has switched. (the back/forward/reload option block moved to the top). 
Is this something upstream/in qute itself, or could this be something caused by changes on my system?

**Comment by The-Compiler** [comment, marked as answer]
That's upstream in QtWebEngine, qutebrowser currently doesn't do anything related to the context menu (also see #349).

**Comment by tomiesz** [comment]
Yeah, saw that issue, just wondered if it could've been something on my system somehow. Thanks for the quick answer.
